test/cache_consistency_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+# In general we do not want to expose the innter workings of the caching and unit systems as part of the API.
+# But we do want to be able to test that there is no conflict between what's in the cache file and what is defined
+# in the unit systems. So we compromise by reaching into implementation to force compare them.
+class Measured::CacheConsistencyTest < ActiveSupport::TestCase
+  measurable_subclasses.each do |measurable|
+    if measurable.unit_system.cached?
```

Comment:
💞

Reply:
Nitpick: I'd use `select` followed by `each` here so we don't indent the whole block.